### PR TITLE
More rules to sign up password

### PR DESCRIPTION
**Pull request by robjaq**
Issue: Add more rules to the sign up password #132

Hello! What do you think of this proposed solution for adding more rules? I still need to write tests for it though but I tried to implement most as pure functions.

### CODE REVIEW

File: iOS-Email-Client/Libs/ValidateString.swift [hunk 1]
```diff
@@ -0,0 +1,143 @@
...
+    if (errorCount == count) {
+      errors.append(error)
+    }
+  })
+  return errors
+}
+
+struct Validators {
+  static func createSignUp() -> ValidateString {
+//    let signUpCharRules = apply(rules: [hasUppercase,
```
Comment (jorgeblacio) [inline iOS-Email-Client/Libs/ValidateString.swift]: Do not leave commented code.

Comment (robjaq) [reply iOS-Email-Client/Libs/ValidateString.swift]: What rules would you like the password to fulfill?

Comment (jorgeblacio) [reply iOS-Email-Client/Libs/ValidateString.swift]: The only rule we are using right now for password on the signup is that it should be at least 8 characters long and match the password repeat field. No other restrictions are in place.

Comment (robjaq) [reply iOS-Email-Client/Libs/ValidateString.swift]: Sounds good

**Pull request closed without merging**